experimental/dds/tree2/src/test/rebaserAxiomaticTests.ts

```diff
@@ -48,11 +50,15 @@ export function runExhaustiveComposeRebaseSuite<TContent, TChangeset>(
 
 	function rebaseTagged(
 		change: TaggedChange<TChangeset>,
-		...baseChanges: TaggedChange<TChangeset>[]
+		rebasePath: TaggedChange<TChangeset>[],
+		baseBranch: TaggedChange<TChangeset>[],
 	): TaggedChange<TChangeset> {
 		let currChange = change;
-		const metadata = defaultRevisionMetadataFromChanges([change, ...baseChanges]);
-		for (const base of baseChanges) {
+		const metadata = rebaseRevisionMetadataFromInfo(
+			defaultRevInfosFromChanges(rebasePath),
+			baseBranch.map(({ revision }) => revision),
+		);
+		for (const base of rebasePath) {
 			currChange = tagChange(rebase(currChange.change, base, metadata), currChange.revision);
```

Comment:
We have to create a separate `RebaseRevisionMetadata` for each step of the rebase. See example here: https://github.com/microsoft/FluidFramework/blob/ea326458fe2ce16d746149ecc220447beb86cdd8/experimental/dds/tree2/src/test/feature-libraries/sequence-field/utils.ts#L141